components/Press/PressLinks/__tests__/PressLinks.test.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import createShallowSnapshotTest from 'test-utils/createShallowSnapshotTest';
+import { mount, shallow } from 'enzyme';
+
+import PressLinks from '../PressLinks';
+import LinkGroup from '../LinkGroup';
+
+describe('PressLinks', () => {
+  test('it should render properly no props', () => createShallowSnapshotTest(<PressLinks />));
+});
+
+describe('PressLinks > LinkGroups', () => {
+  test('it should render properly with required props', () =>
+    createShallowSnapshotTest(
+      <LinkGroup
+        title="test"
+        links={[{ url: 'https://example.com', title: 'Example' }]}
+        MaxLinks={1}
+      />,
+    ));
+
+  test('it should render a button', () =>
```

Comment:
In reality, this isn't testing whether or not a button is being rendered. I think that's a great test case to have though - you mind fleshing out a more targeted test? Additionally, I'd wager you could justify having a second snapshot test and call it: "should render properly with required props and 3 links"